docs/source/DeveloperGuide/Systems/Framework/stableconns/service.py

```diff
@@ -1,15 +1,20 @@
-import types
-from DIRAC import S_OK, S_ERROR
+""" This is a simple example of implementation of a Ping/Pong service for executors.
+
+    This service does not any specific configuration to run.
```

Comment:
does not have/need ? 
I would maybe still put as an example the Port parameter of the configuration